app/views/stats/index.html.erb

```diff
@@ -74,25 +77,36 @@
           <li>
             <dl>
               <dt>
-                <%= link_to work.title, work_path(:id => work.id) %> 
-                <% if params[:flat_view] %><span class="fandom">(<%= work.fandom_string %>)</span><% end %>
-                <span class="words">(<%= ts("%{wordcount} words", wordcount: number_with_delimiter(work.word_count)) %>)</span>
+                <%= link_to work.title, work.item_path %>
+                <% if work.series? %>(Series)<% end %>
+                <% if (params[:view_type] != nil && params[:view_type] != "fandom") %><span class="fandom">(<%= work.fandom_string %>)</span><% end %>
+                <% if work.work? %><span class="words">
+                  (<%= t(".word_count_with_input", wordcount: number_with_delimiter(work.word_count)) %>)</span>
+                <% end %>
               </dt>
               <dd>
                 <dl class="stats">
-                  <% if (work_subscriber_count = Subscription.where(:subscribable_id => work.id, :subscribable_type => 'Work').count) > 0 %>
-                    <dt class="subscriptions"><%= ts("Subscriptions:") %></dt>
-                    <dd class="subscriptions"><%= number_with_delimiter(work_subscriber_count) %></dd>
+                  <% if work.series? %>
+                    <dt class="work_count"><%= t(".work_count") %>:</dt>
```

Comment:
The class should be plain `works` to correspond with what we use on series blurbs.